analyzers/tests/SonarAnalyzer.UnitTest/AnalysisContext/SonarAnalysisContextTest.ShouldAnalyzeGenerated.cs

```diff
@@ -130,7 +156,7 @@ private static Mock<AdditionalText> MockAdditionalText(SourceText sourceText, st
     }
 
     private static Compilation CreateDummyCompilation(AnalyzerLanguage language) =>
-        TestHelper.Compile(string.Empty, false, language).Model.Compilation;
+        SolutionBuilder.Create().AddProject(language).AddSnippet(string.Empty, "NormalFile" + language.FileExtension).GetCompilation();
```

Comment:
It would be great to update `OtherFileName` to not have the dot. Then you could use the same variable here and, as a side effect, the file extensions in the tests will contain the dot which would make the code a little bit easier to read.